BUILD.gn

```diff
@@ -373,66 +373,84 @@ action("create_symbols_dist") {
 action("create_dist_zips") {
   output = "$brave_dist_dir/$brave_product_name-v$brave_version-$brave_platform-$target_arch.zip"
 
-  script = "//brave/script/create-dist.py"
+  if (is_win) {
+    # Repack a chrome release archive to the brave format.
+    # TODO(atuchin): support other platforms.
 
-  inputs = get_target_outputs(":brave_license_dist_resources")
+    script = "//brave/script/repack-archive.py"
+    outputs = [ output ]
 
-  if (!is_mac && !is_android && !is_ios) {
-    inputs += get_target_outputs(":brave_dist_resources")
-    inputs += get_target_outputs(":brave_locale_dist_resources")
-    if (enable_extensions) {
-      foreach(locale, locales) {
-        inputs += get_target_outputs(":brave_shields_locales_${locale}")
-        inputs += get_target_outputs(":brave_rewards_locales_${locale}")
+    deps = [
+      ":create_symbols_dist",
+      "//chrome/installer/mini_installer:mini_installer_archive",
+    ]
+    target_dir = "Chrome-bin"
+    input = rebase_path("$root_out_dir/chrome.7z")
```

Comment:
some args are initialized and then appended, some args are inserted directly into the `args` array below.

Reply:
Rewrite this section a little bit.
inputs is a special for gn, better to preserve it.